game/start_game.esc

```diff
@@ -1,14 +1,13 @@
 :init
 
 set_sound_state _music res://game/sfx/Game-Menu_Looping.mp3 true
-spawn _main_menu res://addons/escoria-core/ui_library/menus/main_menu/main_menu.tscn false
-set_active _main_menu true
+
+# Showing main menu with automatic transition DISABLED
+show_menu main false
```

Comment:
Please remove the "false" here as discussed on Discord

Reply:
Done